File: src/main/java/me/taubsie/dungeonhub/application/enums/EmbedColor.java
package me.taubsie.dungeonhub.application.enums;

import java.awt.*;

public enum EmbedColor {
    POSITIVE(new Color(0, 255, 0)),
    NEGATIVE(new Color(255, 0, 0)),
    INFORMATION(new Color(255, 255, 255)),
    DEFAULT(new Color(165, 23, 112));

    private final Color color;

    EmbedColor(Color color) {
        this.color = color;
    }

    public Color getColor() {
        return color;
    }
}

File: src/main/java/me/taubsie/dungeonhub/kord/application/enums/EmbedColor.kt
package me.taubsie.dungeonhub.kord.application.enums

import java.awt.Color

//TODO update names like Positive to better use kotlin's style
enum class EmbedColor(internalColor: Color) {
    POSITIVE(Color(0, 255, 0)),
    NEGATIVE(Color(255, 0, 0)),
    INFORMATION(Color(255, 255, 255)),
    DEFAULT(Color(165, 23, 112));

    val color: dev.kord.common.Color = dev.kord.common.Color(internalColor.red, internalColor.green, internalColor.blue)
}